src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${(age)}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {(isMarried)
+          ? `${(sex === 'm')
+            ? `${partnerName} is my wife`
```

Comment:
```suggestion
            ? `${partnerName} is my ${(sex === 'm') ? ... : ...}`
```

can be more simplify